lucet-concurrency-tests/tests/killswitch.rs

```diff
@@ -0,0 +1,3 @@
+use lucet_concurrency_tests::killswitch_tests;
+
+killswitch_tests!(lucet_runtime::MmapRegion);
```

Comment:
How best to represent the work of doing this same line for UserfaultFd?  Perhaps an issue?  Perhaps a comment in https://github.com/bytecodealliance/lucet/pull/492?

Reply:
- [ ] If this lands before #492, add a comment there to copy `killswitch_tests!()`

:grin: